scss/_patterns_navigation-reduced.scss

```diff
@@ -0,0 +1,120 @@
+@mixin vf-p-navigation-reduced {
+  @media (min-width: $breakpoint-navigation-threshold - 1) {
+    .p-navigation--sliding.is-reduced.has-search-open {
+      .p-navigation__nav .p-navigation__items {
+        display: inline-flex;
+      }
+
+      .p-navigation__search {
+        align-items: center;
+        background-color: #262626;
+        display: flex;
+        justify-content: center;
+        left: 0;
+        padding: 0.625rem 1rem;
```

Comment:
Where does the 0.625rem value come from? Ideally it should be one of Vanilla predefine spacings (or if there is any calculation required it should be explicit using `calc()` or at least a comment mentioning what is this value).

Overall all spacing (unless it really needs to be custom, should be defined in variables). These are the main ones, but there are some more specific ones in the file as well: 

https://github.com/canonical/vanilla-framework/blob/69b114f64add8e3d25a3acb28c4ca871b5782f8a/scss/_settings_spacing.scss#L79-L93

Reply:
Do you mean all rem values should be in variables, ie. padidng: $spv--small $spv--large $spv--small for example?